### DISCUSSION TITLE
Why deprecating report.extra ? By doing so, you will break the compatibility of existing plugins that override pytest_runtest_makereport hook

### DISCUSSION THREAD

**Discussion by harmin-parra** [pytest-dev/pytest-html#709] [General]
The commit https://github.com/pytest-dev/pytest-html/pull/626 renames the `report` attribute `extra` to `extras`.

By doing so, you break the compatibilities of all existing pytest-html plugins that override the `pytest_runtest_makereport` hook.

Is it really necessary to rename this attribute ?

**Comment by BeyondEvil** [comment]
The `extra` attribute will continue to work (with a deprecation warning) in v4.

Yes, the attribute is a list, and so the name should reflect that.
